tests/cisco_asa/show_running-config_object_network/cisco_asa_show_running-config_object_network.raw

```diff
@@ -2,15 +2,13 @@ object network corp
  subnet 10.0.0.0 255.0.0.0
 object network internal
  subnet 192.168.0.0 255.255.255.0
-object network any
- subnet 0.0.0.0 0.0.0.0
 object network host-10.10.11.1
  host 10.10.11.1
 object network block
  host 10.75.51.11
 object network dmz
- description dmz GW ip
```

Comment:
Why is this being adjusted?

Reply:
The description of the object goes to the next line (after the host)

After:

```
object network dmz
 host 11.1.2.2
 description dmz GW IP
```

Before:
```
object network dmz
  description dmz GW IP
  host 11.1.2.2
```